ToopherSecondFactor/ToopherSecondFactor.xml

```diff
@@ -4,66 +4,12 @@
   <Callbacks length="0" order="1" timeout="600" header="Toopher">
   <!-- dummy callback so code can figure out what to do next -->
   </Callbacks>
-  <Callbacks length="1" order="2" timeout="600" header="Pair with Toopher">
+  <Callbacks length="1" order="2" timeout="600" header="Authenticate with Toopher">
     <NameCallback isRequired="true">
-      <Prompt>Please enter a Pairing Phrase generated from the Toopher Mobile app</Prompt>
+      <Prompt>#TOOPHER_HIDE#</Prompt>
```

Comment:
Javascript uses that prompt string to locate the `<input>` element in the form that needs to receive the final Toopher signature once authentication is finished.  It then finds the parent `<form>`, hides it, and replaces it with our iframe.